docs/docs/feature-reference/output-streaming.md

```diff
@@ -0,0 +1,56 @@
+# Live-Streaming Tool Output
+
+For **synchronous** runs with select tools, Toolchest supports streaming tool output live to your terminal.
+
+By setting the argument `streaming_enabled=True`, lines will be printed to your terminal as they would otherwise be 
+printed by the tool to stdout and stderr.
+
+For example, here's a `python3` Toolchest call with streaming enabled and an example script:
+```python
+import toolchest_client as tc
+
+tc.set_key("YOUR_KEY")
+
+tc.python3(
+    script="script.py",
+    streaming_enabled=True,
+)
+```
+
+```python
+# script.py
+import time
+print("==> Begin streamed lines <==")
+for number in range(5):
+    print(number)
+    time.sleep(1)
+print("==> End streamed lines <==")
+```
+
+The Toolchest call will print the following lines:
+```text
+Beginning Toolchest analysis run.
+Found 1 files to upload.
+==> Begin streamed lines <==
+0
+1
+2
+3
+4
+==> End streamed lines <==
+==> End of stream, connection closed by server <==                                                                      
+Running 1 job | Duration: 0:01:20 | 1 job complete  
+Checking output...
+
+Your Toolchest run is complete! The run ID and output locations are included in the return.
+```
+The streamed lines will be printed as they are generated by the remotely-running Python script!
+
+## Suppoted Tools
+Output streaming is supported for `python3` and `lug`. For both, streaming is **enabled** by default. 
+
+## Troubleshooting
+If output streaming is enabled, tool execution will also be tied to the output stream. This means that
+if the streaming connection is broken, tool execution will also stop.
```

Comment:
Lose the first sentence, keep the second. E.g. "With streaming enabled, tool execution terminates if the streaming connection is broken."